### DISCUSSION TITLE
Will you support the android app on Win11 in the future?

### DISCUSSION THREAD

**Discussion by goxia** [microsoft/winget-cli#1219] [Ideas] [answered]
Will you support the android app on Win11 in the future?

**Comment by denelon** [comment, marked as answer]
We don’t currently have support for APKs as an installer type in the Windows Package Manager. Support for them would have to be prioritized against .exe and .zip installers as well as other Bugs and Features we’re working on. We are likely just as excited as you are about the recent announcement. At this time no firm decision has been made. We will provide an update if anything changes.
